.circleci/config.yml

```diff
@@ -130,14 +130,91 @@ jobs:
     working_directory: ~/tiger_data
     machine: true
     resource_class: pulibrary/ruby-deploy
+    parallelism: 4
     environment:
       ARCH: linux
       COVERALLS_PARALLEL: true
       POSTGRES_USER: tiger_data_user
       POSTGRES_DB: test_db
       POSTGRES_HOST_AUTH_METHOD: trust
       RAILS_ENV: test
-      SPEC_OPTS: "--format progress --format RspecJunitFormatter --out /tmp/rspec/rspec.xml"
+      SPEC_OPTS: "--tag ~type:system --tag ~inconsistent --format progress --format RspecJunitFormatter --out /tmp/rspec/rspec.xml"
+    steps:
+      - checkout
+      - cleanup_docker
+      - run_mediaflux
+      - ruby/install:
+          version: "3.3.0"
+      - install_yarn_dependencies
+      - install_dependencies
+      - install_crypto_policies
+      - run_postgres
+      # wait for postgres and mediaflux
+      - run: sleep 10
+      - enable_sha1_for_mediaflux
+      - persist_to_workspace:
+          root: &root "~/tiger_data"
+          paths: "*"
+      - run: bundle exec rake db:migrate RAILS_ENV=test
+      - run:
+          name: Run Rspec
+          command: bundle exec rspec $(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)
+      - disable_sha1
+      - store_test_results:
+          path: /tmp/rspec
+      - store_artifacts:
+          path: /home/circleci/tiger_data/tmp/capybara
+      - coveralls/upload:
+          coverage_reporter_version: "v0.6.17"
+
+  test_inconsistent:
+    working_directory: ~/tiger_data
+    machine: true
+    resource_class: pulibrary/ruby-deploy
+    environment:
+      ARCH: linux
+      COVERALLS_PARALLEL: true
+      POSTGRES_USER: tiger_data_user
+      POSTGRES_DB: test_db
+      POSTGRES_HOST_AUTH_METHOD: trust
+      RAILS_ENV: test
+      SPEC_OPTS: "--tag inconsistent --format progress --format RspecJunitFormatter --out /tmp/rspec/rspec.xml"
+    steps:
+      - checkout
+      - cleanup_docker
+      - run_mediaflux
+      - ruby/install:
+          version: "3.3.0"
+      - install_yarn_dependencies
+      - install_dependencies
+      - install_crypto_policies
+      - run_postgres
+      # wait for postgres and mediaflux
+      - run: sleep 10
+      - enable_sha1_for_mediaflux
+      - persist_to_workspace:
+          root: &root "~/tiger_data"
+          paths: "*"
+      - run: bundle exec rake db:migrate RAILS_ENV=test
+      - run:
+          name: Run Rspec
+          command: bundle exec rspec || true
+      - disable_sha1
+      - store_artifacts:
+          path: /home/circleci/tiger_data/tmp/inconsistent_specs.log
+
+  test_browser:
+    working_directory: ~/tiger_data
+    machine: true
+    resource_class: pulibrary/ruby-deploy
```

Comment:
Test suites which require a headless browser to run are not run in parallel (I could not find any performance increase in my attempts to parallelize these)